lib/packagekit-glib2/pk-progress-bar.c

```diff
@@ -132,7 +133,7 @@ pk_progress_bar_draw (PkProgressBar *self, gint percentage)
 		g_string_append (str, " ");
 	g_string_append (str, "] ");
 	if (percentage >= 0 && percentage < 100)
-		g_string_append_printf (str, "(%i%%)  ", percentage);
+		g_string_append_printf (str, _("(%i%%)  "), percentage);
```

Comment:
Are trailing spaces really needed here?  AFAICS, it's already at the end of the progress line.